sys/mips/include/asm.h

```diff
@@ -223,11 +227,12 @@ _C_LABEL(x):
  * END
  *	Mark end of a procedure.
  */
-#define	END(x)			\
+#define	END(x) \
```

Comment:
There are some upstream places that would need to be fixed I think if this landed upstream.  I had added manual cfi annotations in the MIPS rtld bits that currently have explicit .cfi_endproc for example.  I suppose you would get compile errors when trying to merge upstream though.

Reply:
You get a (rather unhelpful) compiler error which is how I fixed the ones we have in RTLD right now.

Reply:
Seems to work fine upstream (after resolving conflicts): https://reviews.freebsd.org/D27088